data/korath/korath ships.txt

```diff
@@ -404,9 +404,9 @@ ship "Kas'ik Tek 7"
 		"engine capacity" 18
 		"atmosphere scan" 100
 		"outfit scan power" 18
-		"outfit scan speed" 2
+		"outfit scan efficiency" 20
 		"cargo scan power" 6
-		"cargo scan speed" 2
+		"cargo scan efficiency" 10
```

Comment:
Is there a reason that outfit scan speed->efficiency gets a 10x multiplier, but the cargo one gets 5x?

Reply:
for most ships, cargo = smol, outfit = beeg, and that's about it.

Reply:
(Also cargo scan power = 1/3 outfit scan power on the kas'ik tek, so it seemed like it was prioritizing outfits over cargo)